src/components/CheckListItem/CheckItemActionButtons.tsx

```diff
@@ -47,8 +47,10 @@ export const CheckItemActionButtons = ({ check, showViewDashboard, onRemoveCheck
 
   return (
     <ButtonGroup className={styles.actionButtonGroup}>
-      {showViewDashboard && (
-        <Button onClick={() => showDashboard(check, checkType)} size="sm" variant="link">
+      {viewDashboardAsIcon ? (
+        <IconButton name="apps" onClick={showDashboard} />
```

Comment:
Is this attr `name="apps"` actually used anywhere? And should it be `name="dashboard"` or something instead?
I'm still learning my way around, so I might be looking in the wrong place, but I was expecting to find this `name` attr or something similar rendered directly into the DOM, onto this element.